gwt-dom/src/main/java/org/gwtproject/dom/client/MediaElement.java

```diff
@@ -42,39 +42,39 @@
   @JsOverlay public static final String CANNOT_PLAY = "";
 
   /** Constant returned from {@link #getReadyState()}. */
-  @JsProperty(namespace = "HTMLMediaElement")
+  @JsProperty(namespace = JsPackage.GLOBAL, name = "HTMLMediaElement")
```

Comment:
Shouldn't that be `name = "HTMLMediaElement.HAVE_NOTHING"`?

Reply:
ok

Reply:
I wonder, shouldn't all the elements have their `@JsType` reference the true type (here `name = "HTMLMediaElement"`) so all those `@JsProperty` wouldn't need any attribute? (and actually wouldn't even need to be added)

Reply:
The problem with doing `name` is that we currently can duck type objects in GWT2, so we deliberately said that all elements being ported in this way should be marked as name=Object, so that the duck typing still works - the cast is always a noop, as with JSOs.

Definitely not a good idea going forward, but probably useful for existing codebases that may do some cheating here and there.

Reply:
Ack